clevercloud/pre_build.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash -l
 
 git clone "$IC_SECRETS_HTTPS_REPO_URL" secrets-vault
-sops -d secrets-vault/c5/"$IC_ENVIRONMENT".enc.env > .env
+cd secrets-vault || exit
+git checkout alaurent/c5_staging
```

Comment:
À ne pas oublier avant d’intégrer ;)
```suggestion
git checkout master
```